src/Pages/Basket/Basket.js

```diff
@@ -0,0 +1,321 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ItemList from './ItemList/ItemList';
+import QuantityForm from './QuantityForm/QuantityForm';
+import API from '../../config';
+import sum from '../../utils/sum';
+import './Basket.scss';
+
+class Basket extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      items: [],
+      checkedItemsId: [],
+      quantityFormId: '',
+      isCheckedAllItems: true,
+      isClickedCountButton: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchBasketItems();
+  }
+
+  fetchBasketItems = async () => {
+    try {
+      const res = await fetch(`${API}/orders/cart`, {
+        method: 'GET',
+        headers: {
+          Authorization: `${localStorage.getItem('access_token')}`,
+        },
+      });
+
+      const items = await res.json();
+
+      if (items.message === 'SUCCESS') {
+        this.setState({
+          items: items.result,
+          checkedItemsId: items.result.map(item => item.order_item_id),
+        });
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  requestDeleteItem = async orderItemId => {
+    if (orderItemId.length <= 0) {
+      console.error('아이템을 선택하세요');
+      return;
+    }
+
+    try {
+      const res = await fetch(`${API}/orders/cart`, {
+        method: 'PUT',
+        headers: {
+          Authorization: `${localStorage.getItem('access_token')}`,
+        },
+        body: JSON.stringify({
+          order_item_id: orderItemId,
+        }),
+      });
+      const result = await res.json();
+
+      if (result.message === 'DELETE_SUCCESS') {
+        this.deleteItems(orderItemId);
+      } else {
+        console.error('메시지가 올바르지 않습니다');
```

Comment:
```suggestion
        throw new Error('메시지가 올바르지 않습니다');
```
이렇게 에러를 던져야 catch에서 console.error(err) 해놓은 보람이 있을거 같아요

Reply:
네 확인했습니다!